content/en/docs/tasks/configure-pod-container/configure-pod-configmap.md

```diff
@@ -8,44 +8,53 @@ card:
 ---
 
 <!-- overview -->
-ConfigMaps allow you to decouple configuration artifacts from image content to keep containerized applications portable. This page provides a series of usage examples demonstrating how to create ConfigMaps and configure Pods using data stored in ConfigMaps.
-
+{{< glossary_tooltip text="ConfigMaps" term_id="configmap" >}} allow you to decouple
+configuration data from container image artifacts to keep containerized applications
+portable. This page provides a series of usage examples demonstrating how to create
```

Comment:
There are several changes in this page that wrap the text. Is it preferred to wrap the text (say 80 chars) or
leave the longer lines as is?

Reply:
We should encourage wrapping long lines because git works by comparing text line by line, so does github.com.
Wrapping long lines help identify changes, provide review suggestions, track upstream (en) changes when localizing a page.
Wrapping long lines at 80 characters boundary or 120 characters boundary? That doesn't matter too much. Maybe 80 characters is too restrictive or old fashioned?
When I'm editing a paragraph using vim, I always end by selecting at least one character in a paragraph and then press Q. The whole paragraph is well wrapped.

Reply:
Ok, so what I could do is add a first commit that wraps long lines.

Reply:
Done!